CRM/Event/ActionMapping.php

```diff
@@ -158,7 +158,7 @@ public function createQuery($schedule, $phase, $defaultParams) {
     $query['casContactTableAlias'] = NULL;
     $query['casDateField'] = str_replace('event_', 'r.', $schedule->start_action_date);
     if (empty($query['casDateField']) && $schedule->absolute_date) {
-      $query['casDateField'] = $schedule->absolute_date;
+      $query['casDateField'] = "'{$schedule->absolute_date}'";
```

Comment:
this feels wrong - why is it needed?

Reply:
Yeah. This is wrong. Not because of the quoting I added though. We need to quote it because usually casDateField is a column name and gets added to the query raw. What's wrong with this is we need to escape it. I've pushed up a fix where I escape it.

Reply:
I should also note that without the single quotes it causes a crash if you try and run an event scheduled reminder with an absolute date.